compiler/AST/symbol.cpp

```diff
@@ -21,6 +21,7 @@
 #define __STDC_FORMAT_MACROS
 #endif
 
+#include "../../runtime/include/encoding-support.h"
```

Comment:
Hmm... This is OK but as a style matter I tend to avoid paths like this in includes. (Don't know if it presents a C portability problem). How about we add `runtime/include/encoding` or something along those lines and put the .h file in there? Then the compiler Makefile would just add a -I for that path, and this would just say `#include "encoding-support.h"`.